src/TSTransformer/nodes/statements/transformImportEqualsDeclaration.ts

```diff
@@ -31,7 +40,6 @@ export function transformImportEqualsDeclaration(state: TransformState, node: ts
 		return statements;
 	} else {
 		// Identifier | QualifiedName
-		// See: https://github.com/roblox-ts/roblox-ts/issues/1895
```

Comment:
Comment might be valuable to leave in anyways? To explain rationale for the transform

Reply:
Added back the comment :+1: